app/templates/verkiezingen/installatievergadering.hbs

```diff
@@ -63,7 +63,13 @@
         @bestuursorganen={{@model.bestuursorganenInTijd}}
         @sort={{this.sort}}
         @form={{@model.mandatarisNewForm}}
-      />
+      >{{#if (await bestuursorgaan.isBCSD)}}
+          <Verkiezingen::BcsdVoorzitterAlert
+            @bcsdBestuursorgaanInTijd={{bestuursorgaan}}
+            @bestuursorganenInTijd={{@model.bestuursorganenInTijd}}
+          />
+        {{/if}}
```

Comment:
Is it still useful to have this if in the `yield`? Can't it be moved to `PrepareLegislatuurSection` component?

Reply:
Why are we not putting this component in the `PrepareLegislatuurSection` component? Everytime an update is done in the table it reloads and the alert check can be triggered. If we do this we get rid of the polling and it is checked on every update. 

Im I missing somehting do this?